<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>84</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="UserText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>49</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BanButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UnbanButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UNBAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsersHelp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>65</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>users</string>
    </property>
   </widget>
   <widget class="QLabel" name="MessagesHelp">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>123</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>messages</string>
    </property>
   </widget>
   <widget class="QTreeView" name="UsersView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>191</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="MessagesView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>221</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
